dev-site-config/head.html

```diff
@@ -0,0 +1 @@
+<script> console.log("head html script from a file") </script>
```

Comment:
Do we still want this guy?

Reply:
NVM. I see this has been added for site validation of config

Reply:
yeah, it's nice to have examples.